examples/dbus_client.py

```diff
@@ -15,4 +15,5 @@
 objects = object_manager.GetManagedObjects()
 for object_path in blivet.ListDevices():
     device = objects[object_path]['com.redhat.Blivet1.Device']
-    print(device['Name'], device['Type'], device['Size'], device['FormatType'])
+    fmt = objects[device['Format']]['com.redhat.Blivet1.Format']
+    print(device['Name'], device['Type'], device['Size'], fmt['Type'])
```

Comment:
Would be nice to show how to get the format object and some info from it, but that can be added later.